### PR TITLE
Write HCL files inline

### LINKED ISSUE #6: feat: `-i` flag for in-place edits
now that there's a `-f` flag to specify a file, and we don't have to rely on stdin being piped...

(only) ***when*** the `-f` is used, `-i` should enable in-place editing. i.e. write to that file instead of printing the results

### PR DESCRIPTION
**Pull request by jsemond**
Closes #6

### CODE REVIEW

File: src/bin/hq.rs [hunk 1]
```diff
@@ -10,9 +10,12 @@ use hq_rs::{parse_filter, query, write};
 #[derive(Parser)]
 #[command(version, about)]
 struct Args {
-    #[clap(short, long, value_name = "FILE", help = "HCL file to read from")]
+    #[clap(short = 'f', long = "file", value_name = "FILE", help = "HCL file to read from")]
     file: Option<String>,
 
+    #[clap(short = 'i', long = "inline", help = "Write to HCL file inline instead of stdout (--file must also be set)", requires="file")]
+    inline: bool,
```
Comment (miller-time) [inline src/bin/hq.rs]: This should be an option for the `write` command only, not defined at the top level like this

**Review by miller-time — CHANGES_REQUESTED** [review]

File: src/bin/hq.rs (hunk 1 above)
Comment (rcousineau-xandr) [reply src/bin/hq.rs]: tested it out...

```
➜ cp example.hcl copy.hcl

➜ hq -f copy.hcl '.fmt_block.first_formatted_field'
"fmt_value"

➜ hq -f copy.hcl -i '.fmt_block.first_formatted_field' write '"new_value"'

➜ hq -f copy.hcl '.fmt_block.first_formatted_field'
"new_value"
```

works great! just a little awkward to put `-i` so far left of `write`, when I'd expect it to be structured like:

```
hq -f copy.hcl '.fmt_block.first_formatted_field' write -i '"new_value"'
```

**Review by miller-time — APPROVED** [review]
Lovely!

**Comment by miller-time** [comment]
This is available in v0.13.0, thanks @jsemond 😁

**Pull request merged**